resin_release_tool/releaser.py

```diff
@@ -21,6 +24,68 @@ def get_canaries(self):
             if tag['tag_key'] == 'CANARY']
         return canaries
 
+    def get_tags_per_device(self):
+        all_apps = self.models.application.get_all()
+        if len(next((app for app in all_apps if app['id'] == int(self.app_id)),{})) != 0: #checks if the app exists
+            print("Found app with app_id ", self.app_id, "in Balena")
+            app_info = next((app for app in all_apps if app['id'] == int(self.app_id)),{})
+            all_devices = self.models.device.get_all_by_application_id(app_info['id'])
+            tags_per_device = [{'uuid':device['uuid'],
+                                'tags':self.models.tag.device.get_all_by_device(device['uuid'])}
+                               for device in all_devices]
+            entries_to_remove = ('id', 'device', '__metadata')
+            for device in tags_per_device: #removes all the key/values except, "tag_key" and "value"
+                for tag in device['tags']:
+                    for k in entries_to_remove:
+                        tag.pop(k)
+            for device in tags_per_device:
+                dict_of_vars = {}
+                for tag in device['tags']:
+                    dict_of_vars[tag['tag_key']] = tag['value']
+                del(device['tags'])
+                device['tags'] = dict_of_vars
+            self.uuid_list = [uuid['uuid'] for uuid in tags_per_device]
+            return tags_per_device
+        else:
+            print("Could not find the app with app_id ", self.app_id)
+                    
+    def get_app_env_vars(self):        
+        allvars = self.models.environment_variables.application.get_all(int(self.app_id))
+        list_of_app_env_vars = [{'id':var['id'], var['name']:var['value']} for var in allvars]
+        return list_of_app_env_vars
+        
+    def get_device_env_vars(self):
+        list_of_env_vars_per_device = []
+        for device in self.uuid_list:
+            allvars = self.models.environment_variables.device.get_all(device)
+            list_of_vars = [{'id':var['id'], var['name']:var['value']} for var in allvars]
+            list_of_env_vars_per_device.append({device:list_of_vars})
+        return list_of_env_vars_per_device
+
+    def set_device_env_vars_from_tags(self, tags_per_device, device_env_vars):
+        for device in tags_per_device:
+            uuid = device['uuid']
+            print("----------------------------------------------------------")
+            print("Device: ", uuid)
+            try:
+                value = json.dumps(device['tags'])
+                self.models.environment_variables.device.create(device['uuid'], 'DEVICE_TAG_LIST', value)
+                print("creating/overriding var: ", device['uuid'], 'DEVICE_TAG_LIST', value)
+            except:
```

Comment:
I agree

Reply:
Noted, for sure it will require some more refactoring and integrations in the future but for now I think we can merge it since we need to provide the tags to the ops team.
Thanks @dequis @OrangeTux for the review!

Reply:
Yeah the pylint stuff is out of scope.

Reply:
But this isn't resolved, the except is still too broad